speccpu2017/run_speccpu

```diff
@@ -267,26 +294,82 @@ while [[ $# -gt 0 ]]; do
 			shift 2
 		;;
 		--disks)
+			if [ $run_loca_selected -ne 0 ]; then
+				exit_out "Error: --disks run location already designated, cannot designate multiple"
+			fi
+			run_loca_selected=1
+
 			pbench_arg_list="${pbench_arg_list} $1 $2"
-			disks_to_use=${2}
+			disk_to_use=${2}
 			shift 2
 		;;
+		--installed)
+			pbench_arg_list="${pbench_arg_list} $1"
+			installed=1
+			shift 1
+		;;
 		--spec_config)
+			pbench_arg_list="${pbench_arg_list} $1 $2"
 			spec_config=${2}
 			shift 2
-			;;
+		;;
+		--speccpu_iso_file)
+			pbench_arg_list="${pbench_arg_list} $1 $2"
+			speccpu_iso_file=${2}
+			shift 2
+		;;
+		--speccpu_iso_mnt)
+			pbench_arg_list="${pbench_arg_list} $1 $2"
+			speccpu_iso_mnt=${2}
+			shift 2
+		;;
+		--speccpu_run)
+			pbench_arg_list="${pbench_arg_list} $1 $2"
+			speccpu_run=${2}
+			shift 2
+		;;
+		--speccpu_run_dir)
+			if [ $run_loca_selected -ne 0 ]; then
+				exit_out "Error: --speccpu_run_dir run location already designated, cannot designate multiple"
+			fi
+			run_loca_selected=1
+			pbench_arg_list="${pbench_arg_list} $1 $2"
+			speccpu_run=${2}
+			speccpu_run_dir=${2}
+			shift 2
+		;;
+		--use_mfs)
+			if [ $run_loca_selected -ne 0 ]; then
+				exit_out "Error: --use_mfs run location already designated, cannot designate multiple"
+			fi
+			run_loca_selected=1
+			pbench_arg_list="${pbench_arg_list} $1"
+			use_mfs=`df --output=avail,target,source | grep -v Mounted | grep -v tmpfs | sort -n | tail -1 | xargs | sed "s/ /:/g" | cut -d: -f2`
```

Comment:
And then here we can just call it "use_fs" since it doesn't have to mirror the now-replaced use_mfs.